core/templates/dev/head/domain/exploration/ParamChangeObjectFactorySpec.ts

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for ParamChangeObjectFactory.
+ */
+
+import { ParamChangeObjectFactory } from
+  'domain/exploration/ParamChangeObjectFactory';
+
+describe('Param Change Object Factory', () => {
+  let pcof;
+
+  beforeEach(() => {
+    pcof = new ParamChangeObjectFactory();
+  });
+
+  it('should create a param change object from backend', () => {
+    const sampleData = {
+      customization_args: {
+        parse_with_jinja: false,
+        value: '10'
+      },
+      generator_id: 'Copier',
+      name: 'Param change from backend'
+    };
+    const paramChangeObject = (
+      pcof.createFromBackendDict(sampleData));
+
+    expect(paramChangeObject.toBackendDict()).toEqual(sampleData);
```

Comment:
Check the individual values also, as the toBackendDict need not necessarily have the exact same fields as the object.

Reply:
I didn't quite understand. `toBackendDict` will always return `customization_args`, `generator_id` and `name`, the same fields declared in the object.

Reply:
I think this is fine here!